modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusRoutes.scala

```diff
@@ -2,23 +2,45 @@ package org.tessellation.sdk.infrastructure.consensus
 
 import cats.effect.Async
 import cats.syntax.all._
-import cats.{Order, Show}
 
-import org.tessellation.kryo.KryoSerializer
+import org.tessellation.sdk.domain.cluster.services.Cluster
 import org.tessellation.sdk.infrastructure.consensus.registration.RegistrationResponse
 
-import io.circe.{Decoder, Encoder}
+import io.circe.Encoder
 import org.http4s.HttpRoutes
 import org.http4s.circe.CirceEntityCodec._
 import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
 
-class ConsensusRoutes[F[_]: Async: KryoSerializer, Key: Order: Show: Encoder: Decoder](
+class ConsensusRoutes[F[_]: Async, Key: Encoder](
+  cluster: Cluster[F],
+  manager: ConsensusManager[F, Key, _],
   storage: ConsensusStorage[F, _, Key, _]
 ) extends Http4sDsl[F] {
 
-  val routes: HttpRoutes[F] = HttpRoutes.of[F] {
-    case GET -> Root / "consensus" / "registration" =>
+  private val prefixPath = "/consensus"
+
+  private val p2p: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "registration" =>
       storage.getOwnRegistration.map(RegistrationResponse(_)).flatMap(Ok(_))
   }
 
+  private val public: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "info" =>
+      {
```

Comment:
IMO this endpoint returns registered peers in consensus, so it's not a consensus info.
and I'd definitely return the ordinal for which you return these peers, as it may change ordinal by ordinal. What do you think guys @kriscieslak @kpudlik @tbekas @TheMMaciek?